content/en/dynamic_instrumentation/symdb/python.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Enable Autocomplete and Search for Python
+is_beta: true
+private: false
+code_lang: python
+type: multi-code-lang
+code_lang_weight: 10
+---
+{{< beta-callout url="#" btn_hidden="true" >}}
+Autocomplete and search are in public beta.
+{{< /beta-callout >}}
+
+## Requirements
+
+- [Dynamic Instrumentation][1] is enabled for your service.
+- Tracing library [`dd-trace-py`][6] 2.9.0 or higher is installed.
+
+## Installation
+
+Run your service with Dynamic Instrumentation enabled, and additionally enable autocomplete and search:
+
+1. Run your service with Dynamic Instrumentation enabled by setting the `DD_DYNAMIC_INSTRUMENTATION_ENABLED` environment variable to `true`.
+2. Specify `DD_SERVICE` and `DD_VERSION` [Unified Service Tags][5].
+
+Invoke your service:
+
+```shell
+export DD_SERVICE=<YOUR_SERVICE>
+export DD_ENV=<YOUR_ENV>
+export DD_VERSION=<YOUR_VERSION>
+export DD_DYNAMIC_INSTRUMENTATION_ENABLED=true
+export DD_SYMBOL_DATABASE_UPLOAD_ENABLED=true
+ddtrace-run python -m myapp
+```
+
+3. After starting your service with Dynamic Instrumentation and autocomplete and search enabled, you can use Dynamic Instrumentation's IDE-like features on the [**APM** > **Dynamic Instrumentation**][4] page.
+
+## Additional configuration
+
+### Third party detection
+
+If autocomplete suggestions do not appear for your package or module, it may be incorrectly recognized as third-party code. The autocomplete and search features use a heuristic to filter out third-party code, which can sometimes lead to accidental misclassification.
+
+To ensure that your code is properly recognized and to enable accurate autocomplete and search functionality, you can configure the third-party detection settings using the following options:
+
+```
+export DD_THIRD_PARTY_DETECTION_EXCLUDES=<LIST_OF_USER_CODE_MODULES>
+export DD_THIRD_PARTY_DETECTION_INCLUDES=<LIST_OF_ADDITIONAL_THIRD_PARTY_MODULES>
+```
+
+Where a list means a comma separated list of package prefixes, for example
+
+```
+export DD_THIRD_PARTY_DETECTION_EXCLUDES=shopping,database
+```
```

Comment:
I think this should be part of the block above. Otherwise it feels orphaned and you have to explain what it is more fully.

Reply:
I'm not sure I understand, there is a paragraph of text above this, where would I put that?

Reply:
Ohh, I see, that's an example of what you're explaining with the text. I'll leave a new suggestion.

Reply:
Thx, applied!